src/service/service.py

```
@@ -351,6 +385,29 @@ def emit_progress(node_name: str | None) -> None:
+                    # Feature 노드 실행 추적 (크레딧 차감용)
+                    feature_nodes = {"Solve", "Explain", "CreateGraph", "Variant", "Solution", "Check"}
+                    feature_name = node_name
+                    if node_path:
+                        try:
+                            path_str = str(node_path)
+                            if isinstance(node_path, (tuple, list)) and node_path:
+                                feature_name = str(node_path[0])
+                            elif "/" in path_str:
+                                feature_name = path_str.split("/")[0]
+                        except Exception:
+                            pass
```

Ruff (0.14.14)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)